src/component/DesignerLink/DesignerLink.js

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+
+export default function DesignerLink({designer, designerURL}) {
+  return (
+    <>
+      <Link href={designerURL} target="_blank" rel="noreferrer">
+        Designed by <span>{designer}</span>
+      </Link>
+    </>
+  );
+}
+
+const Link = styled.a `
+    text-decoration: none;
+    color: black;
```

Comment:
Just be careful about removing the default styles of anchor elements. There should be some visual indication about it that this element is indeed a link :)